packages/flutter/test_fixes/material.dart

```diff
@@ -554,4 +554,11 @@ void main() {
 
   // Change made in https://github.com/flutter/flutter/pull/100381
   TextSelectionOverlay.fadeDuration;
+
+  // Changes made in https://github.com/flutter/flutter/pull/97972
+  ThemeData themeData = ThemeData();
+  themeData = ThemeData(toggleableActiveColor: Colors.black);
+  themeData = ThemeData.raw(toggleableActiveColor: Colors.black);
+  themeData = ThemeData.copyWith(toggleableActiveColor: Colors.black);
```

Comment:
```suggestion
  ThemeData themeData = ThemeData();
  themeData = ThemeData(toggleableActiveColor: Colors.black);
  themeData = ThemeData(
    toggleableActiveColor: Colors.black,
  );
  themeData = ThemeData.raw(toggleableActiveColor: Colors.black);
  themeData = ThemeData.raw(
    toggleableActiveColor: Colors.black,
  );
  themeData = themeData.copyWith(toggleableActiveColor: Colors.black);
  themeData = themeData.copyWith(
    toggleableActiveColor: Colors.black,
  );
```

Can you add these test cases? The failing customer test is showing that too many commas are being inserted when the dart fix is applied. It appears to be based on formatting - one line or multiline.

 I can't tell if it is a bug with the fix, or a bug with the fix tool itself.

Reply:
Just added :)